src/seaport.ts

```diff
@@ -2017,6 +2024,7 @@ export class OpenSeaPort {
       takerProtocolFee,
       makerReferrerFee,
       waitingForBestCounterOrder: waitForHighestBid,
+      englishAuctionReservePrice: makeBigNumber(englishAuctionReservePrice || 0),
```

Comment:
since `englishAuctionReservePrice` can be undefined in the UnhashedOrder type that you defined below (which i think is better than having it defined for e.g. non-english auctions), i would have this set to `undefined` instead of `makeBigNumber(0)`